src/System.Runtime.WindowsRuntime.UI.Xaml/pkg/System.Runtime.WindowsRuntime.UI.Xaml.pkgproj

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Runtime.WindowsRuntime.UI.Xaml.csproj">
+      <SupportedFramework>net461;netcoreapp2.0;uap10.0.16299;$(AllXamarinFrameworks)</SupportedFramework>
+    </ProjectReference>
+    <ProjectReference Include="..\src\System.Runtime.WindowsRuntime.UI.Xaml.csproj"/>
+
+    <HarvestIncludePaths Include="ref/netcore50" />
+    <HarvestIncludePaths Include="ref/netstandard1.1">
+      <SupportedFramework>netcore45;wp8;net45;netcoreapp1.0</SupportedFramework>
+    </HarvestIncludePaths>
+
+        <!-- we've previously shipped higer netstandard reference assembly version,
+         this is OK since both of these frameworks will unify down to the inbox version -->
+    <ValidatePackageSuppression Include="PermitInboxRevsion">
+      <Value>.NETCore,Version=v4.5.1;WindowsPhoneApp,Version=v8.1</Value>
+    </ValidatePackageSuppression>
+
+    <!-- We can't harvest an assert to more than one path -->
+    <File Include="$(PackagesDir)\System.Runtime.WindowsRuntime.UI.Xaml\4.3.0\runtimes\win8\lib\netstandard1.3\System.Runtime.WindowsRuntime.UI.Xaml.dll">
```

Comment:
This is causing the same issue we hit here:
https://github.com/dotnet/corefx/blob/724ffdee9db1c95314ca3c45f2692c2b7f73d78e/src/System.Runtime.WindowsRuntime/pkg/System.Runtime.WindowsRuntime.pkgproj#L24

The `netstandard1.3` implementation can't be used to satisfy `netcoreapp2.0` since it needs to have a higher version that the live-build `netstandard2.0` ref.